torch/csrc/jit/passes/quantization.cpp

```diff
@@ -782,9 +778,8 @@ bool isWeight(Module& module, Value* v) {
   return result.has_value() ? result.value() : false;
 }
 
-Module getObserverModuleFor(Value* v, const QConfig& qconfig) {
-  return isWeightOfConvOrLinear(v) ? std::get<1>(qconfig)
-                                   : std::get<0>(qconfig);
+Module getObserverModuleFor(Module& m, Value* v, const QConfig& qconfig) {
```

Comment:
can you keep two `isWeight` function for now? although this should still work since we have a valueNeedsToBeQuantized check before calling this function, but we don't really need to go through CallMethod here